conda_index/cli/__init__.py

```diff
@@ -83,6 +83,14 @@
         will keep python 3.8.X and 3.9.Y in the current_index.json, instead of only the very latest python version.
         """,
 )
+@click.option(
+    "--base-url",
+    help="""
+        If packages should be served separately from repodata.json, URL of the
+        directory tree holding packages. Generates repodata_version=2 which is
+        incompatible with existing versions of conda.
+        """,
```

Comment:
Would like to find less awkward wording here.

Reply:
I wonder if we should introduce a `--repodata-version=N` option, and `--base-url` basically implies this, when passed? That make it more explicit what is being generated and help when users migrate?

That also reminds me that we need to add a schema for it to https://github.com/conda/schemas.